packages/dom/src/dom.js

```diff
@@ -638,13 +633,15 @@ export function unwrap( node ) {
 	parent.removeChild( node );
 }
 
+// eslint-disable-next-line valid-jsdoc
 /**
  * Replaces the given node with a new node with the given tag name.
  *
- * @param {Element}  node    The node to replace
- * @param {string}   tagName The new tag name.
+ * @template {keyof HTMLElementTagNameMap} T
```

Comment:
This is the location of where the JSDoc will improve autocompletion and catch typos.

Reply:
> This is the location of where the JSDoc will improve autocompletion and catch typos.

But it's not valid JSDoc ?

Reply:
Not valid per eslint I guess.

Reply:
Where is the `@template` tag to be found? I don't see it anywhere in the standard.

Reply:
It's an extension of jsdoc that is used by the javascript/typescript language service (LSP) that is used by most editors.

You can find more on it [here](https://www.typescriptlang.org/docs/handbook/type-checking-javascript-files.html#template)